clang-tools-extra/docs/clang-tidy/index.rst

```diff
@@ -10,6 +10,7 @@ See also:
    :maxdepth: 1
 
    The list of clang-tidy checks <checks/list>
+   The list of clang-tidy global options <GlobalOptions>
    Clang-tidy IDE/Editor Integrations <Integrations>
    Getting Involved <Contributing>
    External Clang-Tidy Examples <ExternalClang-TidyExamples>
```

Comment:
Just a drive-by comment, noticed this list sometimes uses clang-tidy, sometimes Clang-Tidy.

Reply:
I think the root cause is the style of titles. Some titles uppercase each word some not.
It looks like in clang/docs, most of rst files' titles use uppercase each word.